bin/load_environment_light.sh

```
@@ -15,14 +15,11 @@ sdk install java 17.0.12-zulu
 sdk use java 17.0.12-zulu
 nvm install v20.17.0
 nvm use v20.17.0
-
+echo "Install opfab-cli"
 (
-    echo "Install opfab-cli"
     cd $OF_HOME/cli/src
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)